addons/test_mail/tests/common.py

```diff
@@ -12,6 +12,26 @@ class TestMailCommon(MailCommon):
     def _create_channel_listener(cls):
         cls.channel_listen = cls.env['mail.channel'].with_context(cls._test_context).create({'name': 'Listener'})
 
+    @classmethod
+    def _create_records_for_batch(cls, model, count):
+        records = cls.env[model]
+        partners = cls.env['res.partner']
+        country_id = cls.env.ref('base.be').id,
+        for x in range(count):
+            partners += cls.env['res.partner'].with_context(**cls._test_context).create({
```

Comment:
Could build a list of values and then create to take advantage of create_multi.
Or you especially don't want that for some reason?

Reply:
It was copy pasted. I suppose a create multi could work indeed. Will try.

Reply:
Et bonjour hein.